core/src/com/unciv/ui/worldscreen/PlayerReadyScreen.kt

```diff
@@ -4,24 +4,113 @@ import com.badlogic.gdx.scenes.scene2d.Touchable
 import com.badlogic.gdx.scenes.scene2d.ui.Table
 import com.unciv.Constants
 import com.unciv.ui.images.ImageGetter
+import com.unciv.ui.popup.ToastPopup
 import com.unciv.ui.utils.BaseScreen
+import com.unciv.ui.utils.UncivTextField
 import com.unciv.ui.utils.extensions.onClick
 import com.unciv.ui.utils.extensions.toLabel
-import com.unciv.utils.concurrency.Concurrency
+import com.unciv.ui.utils.extensions.toTextButton
+import java.security.MessageDigest
 
 class PlayerReadyScreen(worldScreen: WorldScreen) : BaseScreen() {
+    var startTurn = true
+    val curCiv = worldScreen.viewingCiv
+    var enteredPassword = curCiv.hotseatPassword
```

Comment:
???? WHAT????

Reply:
I renamed these to be more clear, but I'm not sure if this resolves this conversation :)